### DISCUSSION TITLE
How could I disable the completion animation?

### DISCUSSION THREAD

**Discussion by ofseed** [folke/noice.nvim#250] [Q&A] [answered]
If I toggle code completion in the cmdline prompt, there will be an animation.
I found the animation sometimes annoying because if I switch items too fast, the animation must
be completed so that neovim will be stuck.

https://user-images.githubusercontent.com/61115159/204962765-c86df264-0716-40f5-b7d3-ba903ebce5c4.mp4

**Comment by folke** [comment]
That's not supposed to be that way. There is no animation 😄 

Honestly no idea what could be causing that behavior. I don't think it is Noice tbh...

You seem to be using cmp for auto completion, maybe check there?

**Comment by ofseed** [comment]
This is the most feature-like bug I've ever seen. I'm using cmp-cmdline instead of the popup menu bundled in noice.
But I found that even if I disable cmp-cmdline and enable the popup bundled in noice, and sets the backend to cmp,
This 'feature' still exists. But the nui backend works well. Too weird.

**Reply by nyngwang** [reply]
What a hilarious feature at first look.

**Comment by folke** [comment, marked as answer]
You should check if you have a plugin doing stuff on `CmdlineChanged`.

**Comment by ofseed** [comment]
Thank you for your advice! I found the reason immediately.
It's a plugin called [range-hilight](https://github.com/winston0410/range-highlight.nvim) which causes this issue.

**Reply by nyngwang** [reply]
There are some websites using this as a feature to swap the noun/verb of a sentence. Maybe a good idea to make this into a plugin.

**Reply by folke** [reply]
@nyngwang what do you mean? What does this have to do with op's issue?

**Reply by nyngwang** [reply]
@folke Oops.
